src/templates/iteration/detail.html.twig

```diff
@@ -0,0 +1,151 @@
+{% extends 'base.html.twig' %}
+{% block title %}
+    {{ project.name }}: {{ iteration.start|date("m/d/Y") }} - {{ iteration.finish|date("m/d/Y") }} Iteration Details
+{% endblock %}
+{% block body %}
+    <div class="container">
+        <div class="row the-space-beneath">
```

Comment:
What does `the-space-beneath` mean? :thinking:

Reply:
😂 it is referring to the css 
```css
.the-space-beneath {
    margin-bottom: 20px;
}
```

I couldn't think of a better way, at the time, to explain that I wanted space beneath this HTML element.  I also didn't want to apply the margin to all `row`s.

Without the class:

<img width="1175" alt="Screen Shot 2020-03-02 at 8 38 03 PM" src="https://user-images.githubusercontent.com/2382439/75737607-c9ab9780-5cc5-11ea-9538-b6ca307696dc.png">

----

With the class:
<img width="1161" alt="Screen Shot 2020-03-02 at 8 38 16 PM" src="https://user-images.githubusercontent.com/2382439/75737626-d29c6900-5cc5-11ea-8dd5-ce2f27c0cb2e.png">